package.json

```diff
@@ -130,6 +130,11 @@
           "type":"number",
           "default": 2,
           "description": "When the estimated number of tokens exceeds the limit, binary search will be used to reduce the number of lines. This parameter controls the maximum number of binary search iterations."
+        },
+        "fauxpilot.trimLeftWhitespace": {
```

Comment:
The word `trimLeftWhitespace` should be `trimLeadingWhitespace`.

Reply:
right thanks, fixed :)